dot/state/block_test.go

```diff
@@ -24,17 +24,18 @@ var testGenesisHeader = &types.Header{
 	Digest:    types.NewDigest(),
 }
 
-func newTestBlockState(t *testing.T, header *types.Header, tries *Tries) *BlockState {
+func newTestBlockState(t *testing.T, tries *Tries) *BlockState {
 	ctrl := gomock.NewController(t)
 	telemetryMock := NewMockClient(ctrl)
 	telemetryMock.EXPECT().SendMessage(gomock.Any()).AnyTimes()
 
 	db := NewInMemoryDB(t)
-	if header == nil {
-		header = testGenesisHeader
+	header := types.Header{
```

Comment:
why are we not returning `testGenesisHeader` anymore?

Reply:
Trying to steer away from global variables slowly

Reply:
But it's not intuitive that it still uses `testGenesisHeader` which is asserted on in a bunch of those tests that used to provide `testGenesisHeader` to `newTestBlockState`.

Reply:
Agreed, reverted 👍